locking/gradle.lockfile

```diff
@@ -0,0 +1,70 @@
+# This is a Gradle generated file for dependency locking.
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2021-44832   
**Check Name:** log4j-core: remote code execution via JDBC Appender   
**Severity:** MEDIUM   
**Fixed Version:** 2.12.4, 2.17.1   
**Description:** Apache Log4j2 versions 2.0-beta7 through 2.17.0 (excluding security fix releases 2.3.2 and 2.12.4) are vulnerable to a remote code execution (RCE) attack when a configuration uses a JDBC Appender with a JNDI LDAP data source URI when an attacker has control of the target LDAP server. This issue is fixed by limiting JNDI data source names to the java protocol in Log4j2 versions 2.17.1, 2.12.4, and 2.3.2.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2021-44832